products/laravel.md

```diff
@@ -3,80 +3,97 @@ title: Laravel
 category: framework
 iconSlug: laravel
 permalink: /laravel
+versionCommand: composer show laravel/framework|grep versions
 releasePolicyLink: https://laravel.com/docs/9.x/releases#support-policy
 changelogTemplate: https://laravel.com/docs/__RELEASE_CYCLE__.x/releases
 activeSupportColumn: true
-versionCommand: composer show laravel/framework|grep versions
 releaseDateColumn: true
+
+# Note that laravel/laravel is just a started application.
 auto:
 -   git: https://github.com/laravel/framework.git
+
 identifiers:
 -   purl: pkg:composer/laravel/laravel
 -   repology: php:laravel-framework
 -   purl: pkg:docker/bitnami/laravel
 -   purl: pkg:github/laravel/framework
+
+# Support/EOL dates and supported PHP versions can be found on https://laravel.com/docs/9.x/releases#support-policy
 releases:
 -   releaseCycle: "10"
+    releaseDate: 2023-02-14
     support: 2024-08-06
     eol: 2025-02-04
+    supportedPhpVersions: 8.1 - 8.2
```

Comment:
```
4:20:45 AM:   Liquid Exception: Invalid Date: '"8.1 - 8.2"' is not a valid datetime. in products/laravel.md
4:20:45 AM:              ERROR: YOUR SITE COULD NOT BE BUILT:
4:20:45 AM:                     ------------------------------------
4:20:45 AM:                     Invalid Date: '"8.1 - 8.2"' is not a valid datetime.
4:20:45 AM:                     ------------------------------------------------
4:20:45 AM:       Jekyll 4.3.1   Please append `--trace` to the `build` command 
4:20:45 AM:                      for any additional information or backtrace. 
4:20:45 AM:                     ------------------------------------------------
```

Reply:
Oups, this was because I used the `date` type in the `table.html` declaration. This is fixed.